packages/graphql/src/classes/Node.ts

```diff
@@ -176,6 +160,14 @@ class Node {
     get labels(): string[] {
         return this.nodeDirective?.getLabels(this.name) || [this.name];
     }
+
+    getPlural(camelcase: boolean): string {
```

Comment:
I would personally prefer this to be called using `getPlural({ camelCase: true })`, but it is very much a personal preference thing.

Reply:
I prefer as well, but it collides with the actual camelCase library :laughing: 
I guess we could be more explicit with that fact that this is an option or flag (`camelCaseFlag`)?

Reply:
I was actually talking about the destructuring vs parameter, but funny about the naming conflict!

I think it's just when you see `getPlural(true)` in the codebase, it's unclear what that "magic flag" as you call it is doing. But no real need to change, more of an observation!

Reply:
nah, I though about that, and you are right (already changed it). I think is better to have explicit flags rather than random booleans.

To avoid name conflict I went with an options parameter instead of destructuring